geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/TableConfCommand.scala

```diff
@@ -114,7 +114,7 @@ object TableConfCommand {
     @Parameter(names = Array("-t", "--table-suffix"), description = "Table suffix to operate on (attr_idx, st_idx, or records)", required = true)
     var tableSuffix: String = null
 
-    lazy val ds = new DataStoreHelper(this).getDataStore()
+    lazy val ds = DataStoreParamsHelper.createDataStore(this)
     lazy val tableName = getTableName(ds, this)
```

Comment:
while we're changing this, is ds accessed anywhere else? it should be disposed. i would change to:
```
lazy val tableName = {
  val ds = DataStoreParamsHelper.createDataStore(this)
  val name = getTableName(ds, this)
  ds.dispose()
  name
}
```
you'll need to fix `getTableName` to use the `ds` object that's passed in directly instead of the one from `params`